src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_command_type.py

```diff
@@ -852,6 +854,45 @@ def _load_transformed_arguments(self, handler):
                                              help=docstring))
 
 
+def validate_client_parameters(namespace):
+    """Retrieves Batch connection parameters from environment variables"""
+    from azure.mgmt.batch import BatchManagementClient
```

Comment:
Is this just relocated or is it refactored? If so, might want @annatisch to review.